### PR TITLE
Add error handling in ares-package

### PR DESCRIPTION
**Pull request by ssuminahn**
:Release Notes:
Add error handling in ares-package

:Detailed Notes:
- Add error handling for info and info-detail options of ares-package
- Update indentation in ares-generate.js

:Testing Performed:
1. Pass unit test
2. Pass eslint
3. Check error messages below test case
 - ares-package.js -i
 - ares-package.js -I
 - ares-package.js -i aaa
 - ares-package.js -i aaa.ipk

:Issues Addressed:
[WRN-13892] Add error handling for ares-package

### CODE REVIEW

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/base/error-handler.js [hunk 1]
```diff
@@ -84,12 +84,13 @@ const log = require('npmlog'),
...
             "INVALID_COMMAND" : "This command is invalid. Please check the supported commands using ares -l",
             "INVALID_COMBINATION" : "This options cannot be used with display option",
-            
+
             // set value & type error
             "EMPTY_PROFILE" : "Profile is empty",
             "EMPTY_VALUE" : "Please specify a value",
             "EXISTING_VALUE" : "The specified value already exists",
             "EXISTING_FILETYPE_PATH" : "A file with the same name at the specified location already exists",
+            "SUPPORT_ONLY_IPK" : "Support only .ipk file. Please check the file extension",
```
Comment (YoungeunKim) [inline lib/base/error-handler.js:93]: Why don't you change this error message? 
Only .ipk file is supported.

Please refer to : 
"NOT_SUPPORT_XZ" : "xz file is not supported. Only .ipk file packaged with the CLI is supported",

**Comment by YoungeunKim** [comment]
Verified :+1:

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/package.js [hunk 2]
```diff
@@ -394,8 +394,10 @@ const ar = require('ar-async'),
 
             function _removeTmpDir(next){
                 log.info("package#analyzeIPK()#_removeTmpDir()");
-                fs.rmdirSync(tmpDirPath, {recursive: true, force:true});
-                next();
+                rimraf(tmpDirPath, function(err) {
+                    log.verbose("package#analyzeIPK()#_removeTmpDir()", "removed " + tmpDirPath);
+                    next(err, null);
```
Comment (YoungeunKim) [inline lib/package.js:399]: Could you use "next(err)" instead of next(err, null);

**Review by YoungeunKim — CHANGES_REQUESTED** [review]

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/package.js (hunk 2 above)
Comment (ssuminahn) [reply lib/package.js]: I've updated this.

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

**Review by YoungeunKim — APPROVED** [review]

**Pull request merged**